src/Authoring/WinRT.Host/Directory.Build.targets

```diff
@@ -0,0 +1,28 @@
+<!--
+***********************************************************************************************
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <Target Name="SplitWinRTHostIntoMUI"
+            Inputs="$(OutDir)WinRT.Host.dll"
+            Outputs="$(OutDir)WinRT.Host.dll;$(OutDir)WinRT.Host.dll.mui"
+            AfterTargets="CopyFilesToOutputDirectory">
+
+        <!-- Invoke the muicrt tool to split Winrt.Host into language neutral parts and localizable resources
+            Then invoke the tool again to apply the checksum from the neutral part to the mui
+            Finally, replace the initially built WinRT.Host with the language neutral version -->
+        <PropertyGroup> 
+            <WinRTHostMuirctCommand>Muirct.exe -q "$(MSBuildThisFileDirectory)WinRTHost.rcconfig.xml" -v 2 -x 0x0409 -g 0x0409 "$(OutDir)WinRT.Host.dll" "$(OutDir)WinRT.Host_LN.dll" "$(OutDir)WinRT.Host.dll.mui"
+muirct.exe -c "$(OutDir)WinRT.Host_LN.dll" -e "$(OutDir)WinRT.Host.dll.mui"
+copy /y "$(OutDir)WinRT.Host.dll" "$(OutDir)WinRT.Host.Original.exe"
```

Comment:
original.dll?  but it probably also doesn't matter.

Reply:
Just a temp name, we're going to replace the output WinRT.Host with the language-neutral version anyway